pkg/controllers/user/helm/common.go

```diff
@@ -163,7 +175,11 @@ func (l *Lifecycle) generateTemplates(obj *v3.App) (string, string, *common.Helm
 	}
 
 	// notes.txt
-	commands = append([]string{"template", appDir, "--name", obj.Name, "--namespace", obj.Spec.TargetNamespace, "--notes"}, setValues...)
+	if obj.Spec.HelmVersion != "helm_v3" {
+		commands = append([]string{"template", appDir, "--name", obj.Name, "--namespace", obj.Spec.TargetNamespace, "--notes"}, setValues...)
+	} else {
+		commands = append([]string{"install", obj.Name, appDir, "--dry-run", "--namespace", obj.Spec.TargetNamespace, "--kubeconfig", tempDir.KubeConfigInJail}, setValues...)
+	}
 	cmd = exec.Command(helmName, commands...)
```

Comment:
Does helmName need to be "rancher-helm" for invoking this command as well, when obj.Spec.HelmVersion != "helm_v3"